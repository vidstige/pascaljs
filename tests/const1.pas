program Sample;
const
  I = 123;
  S = 'hej';
begin
  WriteLn('I = ', I);
  WriteLn('S = ', S);
end.