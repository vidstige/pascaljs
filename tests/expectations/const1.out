I = 123
S = hej
